### PR TITLE
Stopped the pluralization of collection name

### PR DESCRIPTION
**Pull request by damienvdw**
**- Summary**

Removed pluralization of collection names in the CMS.

As suggested, this is the minimalist fix for #422

**- Test plan**

Pass all current tests and no additional tests required.

**- Description for the changelog**

Stopped the pluralization of collection name

### CODE REVIEW

File: src/components/UnpublishedListing/UnpublishedListing.js [hunk 1]
```diff
@@ -82,7 +81,7 @@ class UnpublishedListing extends React.Component {
                 <div className={styles.draggable}>
                   <Card className={styles.card}>
                     <UnpublishedListingCardMeta
-                      meta={capitalize(pluralize(collection))}
+                      meta={pluralize(collection)}
```
Comment (erquhart) [inline src/components/UnpublishedListing/UnpublishedListing.js]: I think you meant to remove `pluralize` and leave `capitalize`.

**Comment by erquhart** [comment]
@damienvdw thanks! Just one change needed, otherwise LGTM.

**Comment by damienvdw** [comment]
Wow, I really need more coffee.

It's fixed now 😄

**Comment by damienvdw** [comment]
Yeah, first open source PR ever ! 🎉

**Comment by Benaiah** [comment]
@damienvdw thanks, and congratulations! 🎉

**Comment by ahoward** [comment]
a really good fix.  no code is good code.

**Pull request merged**